Extensions/TileMap/helper/src/tiled/TiledLoaderHelper.ts

```diff
@@ -91,6 +91,8 @@ export const extractTileUidFlippedStates = (
   );
 
   return {
+    //TODO: Fix this error
```

Comment:
same here, it's hard to approve a ts-ignore without knowing what's behind.
why is this needed?